frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -72,6 +72,34 @@ <h5 class="fw-light">Challenge Settings</h5>
       </div>
     </div>
 
+    <!-- Evaluations and Annoatations -->
+    <div class="row settings-section">
+      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-light fs-18">Evaluation and Annoatations:</span>
+      </div>
+      <div class="ev-action">
+        <div *ngIf="isChallengeHost" class="col-sm-1.5 col-xs-1.5 ev-upload-action">
+          <!-- Evaluations Criteria Description -->
+          <span class="fs-16 fw-light">
+            <strong class="text-light-black">Evaluation Criteria: </strong>
+          </span>
+          <a class="pointer fs-16" (click)="editEvaluationCriteria()">
+            &nbsp;&nbsp;
+            <i class="fa fa-pencil" aria-hidden="true"> </i>&nbsp;Edit
```

Comment:
let's remove the `Edit` text as it's already understood from the icon and for consistency on the page as well.

Reply:
Sure. Removed the `Edit` text